packages/shared/src/components/modals/bookmark/BookmarkFolderModal.tsx

```diff
@@ -80,8 +80,8 @@ const PlusCTA = ({ folderCount }: { folderCount: number }) => {
 };
 
 const ModalTitle = () => (
-  <>
-    <ModalHeader.Title>New Folder</ModalHeader.Title>
+  <span className="flex flex-row items-center gap-1 tablet:gap-2">
```

Comment:
But wouldn't it work if you just did:
`<ModalHeaderTitle kind={blabla}>`?

Reply:
But ModalHeaderTitle is just a dummy element created with `classed` 👀 

```tsx
const ModalHeaderTitle = classed('h3', 'font-bold');
```

Reply:
Classes are applied from `ModalHeader` here:

```tsx
{!!modalTitle && (
        <ModalHeaderTitle className={headerKindToTitleClassName[kind]}>
          {modalTitle}
        </ModalHeaderTitle>
      )}
```

Reply:
Ohhhh I see, sorry didn't really dive deep, let's rather keep what you had then sorry 🙏 
Not ideal as it's not verbose, but ok for now.

Reply:
So we need to revert to just adding the className?

Reply:
Yeah sorry for that, think that's cleaner overall

Reply:
NP is fine, at least we have checked different ways to do it ✔️